src/messenger-shortcuts.js

```diff
@@ -115,7 +139,11 @@ function focusSearchBar() {
 }
 
 function focusMessageInput() {
-  document.querySelector('div[role="main"] div[role="textbox"]').click();
+  targetNode = getByAttr('div', 'aria-label', 'Type a message...')
+  triggerMouseEvent (targetNode, "mouseover");
```

Comment:
For some reason, the original implementation with the click() never worked for me.